lib/config/optionsconfiguration.go

```diff
@@ -152,3 +154,20 @@ func (opts OptionsConfiguration) GlobalDiscoveryServers() []string {
 	}
 	return util.UniqueTrimmedStrings(servers)
 }
+
+func (opts OptionsConfiguration) EffectiveMaxConcurrentFolders() int {
+	// If a value is set, trust that.
+	if opts.MaxConcurrentFolders > 0 {
+		return opts.MaxConcurrentFolders
+	}
```

Comment:
I guess you want people to set it to 999 for "unlimited"?

Reply:
I was intending -1 to mean unlimited as usual, but I see know that I didn't implement that.

Reply:
✅ fixed